content/source/docs/enterprise/private/data-security.html.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Security"
+sidebar_current: "docs-enterprise2-private-data-security"
+---
+
+# Private Terraform Enterprise - Security
+
+Private Terraform Enterprise (PTFE) takes the security of the data it manages
+seriously. This table will provide reference for the parts of the PTFE app that can contain sensitive data, what storage is used, and what encryption is used.
```

Comment:
No need for future tense here. "This table lists which parts of the PTFE app can contain..."